main.css

```diff
@@ -0,0 +1,465 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url("https://fonts.cdnfonts.com/css/Lato");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.top-links-container {
+  flex-direction: row;
+  width: 100%;
+  height: 40px;
+  background-color: #272a31;
+}
+
+.top-links {
+  align-items: center;
+  justify-content: end;
+  font-family: 'Lato', sans-serif;
+  padding: 10px 55px 0 0;
+}
+
+li a {
+  text-decoration: none;
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-size: 1rem;
+}
+
+.top-links > a {
+  float: right;
+  color: #d3d3d3;
+  text-decoration: none;
+  margin: auto 5px;
+}
+
+header {
+  background-color: #fff;
+  font-family: 'Lato', sans-serif;
+}
+
+nav {
+  padding: 20px 0;
+}
+
+.nav-toggle {
+  position: absolute;
+  top: 0.5rem;
+  left: 1rem;
+  display: none;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 20px;
+  padding: 0 1rem;
+}
+
+.campain a {
+  border: 3px solid red;
+  padding: 5px 12px;
+  color: gray;
+}
+
+.disable a {
+  color: gray;
+}
+
+.red a {
+  color: red !important;
+}
+
+.partners img {
+  width: 150px;
+  height: auto;
+}
+
+.cards img {
+  width: 100%;
+}
+
+.speakers img {
+  width: 40%;
+  height: auto;
+  margin-right: 10px;
+  box-shadow: -5px -5px 3px rgb(236 82 66 / 30%);
+}
+
+.speak-element {
+  height: auto;
+  margin: 10px auto;
+}
+
+.speak-element > p {
+  width: 100%;
+}
+
+.partners-icon img {
+  width: 150px;
+}
+
+.container h2 a img {
+  text-decoration: none;
+  width: 40px;
+  height: 40px;
+}
+
+ul {
+  padding: 0;
+}
+
+li {
+  display: inline-block;
+  padding: 0 1rem;
+}
+
+.open {
+  height: 100vh;
+  background-color: rgb(275, 275, 275, 0.3);
+  width: 100%;
+  font-size: 1.4rem;
+}
+
+.hidden {
+  display: none;
+}
+
+.sign {
+  margin-top: 20px;
+}
+
+.sign h3 {
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.sign h3,
+h5 {
+  color: #fff;
+}
+
+.main {
+  background-image: url(./components/food2.jpg);
```

Comment:
You have done a great job at customizing the content of the website. Kindly ensure that you follow the design guidelines while doing so.

- The pages are expected to look almost identical to the original design, outside the content differences.
- The pages are expected to use the exact same colors, for text, backgrounds, borders, etc, from the original design.

The hero page of the design has a light background. You could achieve a similar design by having a white overlay. Please, also avoid using an image having an advert watermark (e.g "Adobe Stock" as we have now).

Your design:
![image](https://user-images.githubusercontent.com/20554583/197339037-6acfcda9-c442-4903-93d2-c5f647702114.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/197339065-7ad9d4a5-5b08-4632-9cbf-13639fe81f1c.png)